### PR TITLE
Some Java cleanup: removal of unused imports, addition of some @NotNulls

### PR DESCRIPTION
**Pull request by jwren**

### CODE REVIEW

**Comment by jwren** [comment]
@pq How do I have a bot re-run?

**Comment by pq** [comment]
I kicked it for you.  (You may not have access.)

File: src/io/flutter/sdk/FlutterSdkManager.java [hunk 1]
```diff
@@ -110,23 +110,23 @@ default void flutterSdkRemoved() {
   // Listens for changes in Flutter Library configuration state in the Library table.
   private final class LibraryTableListener implements LibraryTable.Listener {
     @Override
-    public void afterLibraryAdded(Library newLibrary) {
+    public void afterLibraryAdded(@NotNull Library newLibrary) {
```
Comment (pq) [inline src/io/flutter/sdk/FlutterSdkManager.java]: Is this the API as of our lowest bound (`171.1`)?

Comment (pq) [inline src/io/flutter/sdk/FlutterSdkManager.java]: (I assume the bots will tell us?)

Comment (jwren) [inline src/io/flutter/sdk/FlutterSdkManager.java]: I was curious, if the bots would catch it (I guessed they wouldn't)... These @NotNulls have been removed as well.  As soon as the Flutter plugin is on 2017.2.x, this cleanup can happen.

**Comment by pq** [comment]
Thanks!

**Pull request merged**